modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+Total Failed Inbound Requests::
+The sum of failed inbound requests in the period, grouped by application in automatic intervals.
+
+Total Failed Outbound Requests::
+The sum of failed outbound requests in the period, grouped by application in automatic intervals.
+
+Total Inbound Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
+
+Total Outbound Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
+
+Failed Inbound Requests: Response Time by Endpoint::
+The average, maximum, and minimum response times of failed inbound requests, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Total Failed Inbound Requests by Endpoint::
+The sum of failed inbound requests, grouped by endpoint.
+
+Failed Outbound Requests: Response Time by HTTP endpoint::
+The average, maximum, and minimum response times of failed outbound requests, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Total Failed Outbound Requests by HTTP Endpoint::
+The sum of failed outbound requests, grouped by endpoint.
+
+[[jvm-charts]]
+== JVM
+
+The following charts appear on the *JVM* dashboard:
+
+Garbage Collection Count::
+The number of collections that have occurred, grouped by worker in automatic intervals.
+
+Garbage Collection Time::
+The approximate accumulated collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
+
+Par New Collection Count::
+The number of collections for Parnew GC, grouped by type of response in automatic intervals.
+
+Par New Collection Time::
+The approximate accumulated Parnew collection elapsed time in milliseconds, grouped by type of response in automatic intervals.
```

Comment:
... ParNew garbage collection ...

Remove "in milliseconds" 
Remove "grouped by"

Reply:
```suggestion
The average amount of ParNew garbage collection time, grouped by worker in automatic intervals.
```